contracts/HundredVesting.sol

```diff
@@ -6,9 +6,16 @@ import "@openzeppelin/contracts/access/Ownable.sol";
 import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
 import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
 
+// no Ownable modifiers are used
+// does it really need to inherit
+// from Ownable ?
 contract HundredVesting is Ownable {
     IERC20 public immutable Hundred;
     uint256 totalPeriod = 365 * 24 * 60 * 60;
+    // timePerPeriod seems to be used once
+    // in a formulae where its value is canceled
+    // after expanding the expression
+    // which indicate it is not needed
```

Comment:
I believe it's needed. Without it, they can claim tokens by sec.
But with it, we can set the period like one day. Then they can get the claimable tokens every day.